modules/nf-core/last/dotplot/environment.yml

```diff
@@ -1,7 +1,9 @@
-name: last_dotplot
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "last_dotplot"
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
-  - bioconda::last=1453
+  - "bioconda::last=1542"
```

Comment:
Any reason for the quote marks here? Other modules don't have them.

Reply:
To modernize the modules I deleted the files and ran `nf-core modules create` from scratch. This is where the quotes come from.